main/main_builders.py

```diff
@@ -70,7 +70,7 @@ def make_default_controller_mappings(target, source, env):
 
     g.write("/* THIS FILE IS GENERATED DO NOT EDIT */\n")
     g.write("#include \"core/typedefs.h\"\n")
-    g.write("#include \"main/default_controller_mappings.h\"\n")
+    g.write("#include \"core/input/default_controller_mappings.h\"\n")
```

Comment:
I think controller mappings are gone, so the function to make controller mappings should be gone from here too

Reply:
So you mean, you want me to delete everything about controller mapping? (code generator and controllers db txt)

And this:
https://github.com/godotengine/godot/blob/8874de089896cefcccd0a85b24d0b67ed86b6df4/core/input/input_filter.cpp#L1296-L1301
Because I followed the pipe, and I need to delete this too.